plugins/plugin_compile/build_android.py

```diff
@@ -411,6 +411,57 @@ def gradle_build_apk(self, build_mode):
         cmd = '"%s" --parallel --info assemble%s' % (gradle_path, mode_str)
         self._run_cmd(cmd, cwd=self.app_android_root)
 
+    def _do_get_build_type(self, str):
+        """
+        return  -1: doesn't include any build arch information
+                1: only build 64bit
+                2: only build 32bit
+                3: build 64bit and 32bit
```

Comment:
I found a way to implement a simple enum in python. 
You could simply define it like the following:
```python
class Stationary:
    (Pen, Pencil, Eraser) = range(0, 3)
print Stationary.Pen
```